packages/components/src/theme/panel.scss

```diff
@@ -37,5 +37,9 @@
         .panel-content-caption {
             font-weight: 300;
         }
+
+        .panel-content-context > a {
```

Comment:
I see what you mean with regards to this being a less than desirable change. I would say let's forego this styling. It's a bit presumptuous that all anchor tags will want this offset (e.g. a button can also use a `<a/>`). If we really want it for this page we can make a workflow-specific style that doesn't need to change `panel-content-context` at all.

Reply:
I moved this to workflow.  It looks weird in the UI without it